code/datums/components/crafting/recipes/carnival_silk/armor_weaving/carnival_blade_lineage.dm

```diff
@@ -0,0 +1,33 @@
+//Blade Lineage Recipes
+/datum/crafting_recipe/blade_lineage_salsu
+	name = "Blade Lineage Salsu Robe"
+	result = /obj/item/clothing/suit/armor/ego_gear/city/blade_lineage_salsu
+	reqs = list(/obj/item/stack/sheet/silk/steel_simple = 3,
+				/obj/item/stack/sheet/silk/amber_advanced = 1,
+				/obj/item/stack/sheet/silk/human_simple = 1)
```

Comment:
While we do have quite a lot of shitcoded lists, please dont contribute towards them
If the list has 2-3 short items, put it in a single line
if it has 2+ fairly long items, put it in a multi-line like this. (each object being on its own line, this is better for future github additions, removals and changes)
```suggestion
	reqs = list(
		/obj/item/stack/sheet/silk/steel_simple = 3,
		/obj/item/stack/sheet/silk/amber_advanced = 1,
		/obj/item/stack/sheet/silk/human_simple = 1,
	)
```

Reply:
Oh, I guess I didn't really notice how shitty it looked. I will double check all of my lists today.

Reply:
All of them should be fixed now.